deluge/tests/test_torrent.py

```diff
@@ -73,12 +73,24 @@ def get_torrent_atp(self, filename):
         filename = common.get_test_data_file(filename)
         with open(filename, 'rb') as _file:
             info = lt.torrent_info(lt.bdecode(_file.read()))
-        atp = {'ti': info}
-        atp['save_path'] = os.getcwd()
-        atp['storage_mode'] = lt.storage_mode_t.storage_mode_sparse
-        atp['add_paused'] = False
-        atp['auto_managed'] = True
-        atp['duplicate_is_error'] = True
+        if lt.is_version_1_1:
+            atp = {
+                'ti': info,
+                'save_path': os.getcwd(),
+                'storage_mode': lt.storage_mode_t.storage_mode_sparse,
+                'add_paused': False,
```

Comment:
Instead of testing for 1.2. Replace these three dict with the following that should work for both versions:

```
atp['flags']  = (
    LT_DEFAULT_ADD_TORRENT_FLAGS  
    | lt.add_torrent_params_flags_t.flag_duplicate_is_error
    & ~lt.add_torrent_params_flags_t.flag_paused
)
```